### DISCUSSION TITLE
Not able to delete projects (beta)

### DISCUSSION THREAD

**Discussion by mgks** [community/community#6813] [Projects and Issues] [answered]
I suppose every new feature must come with atleast two options if nothing, "Create" and "Delete". But there is no option to delete new beta projects.
Anyway, if I have missed the delete option somewhere in the menu? Please update here, other than that, there is just close project option.

**Comment by timoyates** [comment]
Hi @mgks, thanks for the feedback! This is on our list for the future - I don't have a timeline to share at the moment but rest assured we will indeed get to it 😊 Cheers!

**Reply by mgks** [reply]
Just hire me please, I'll do it myself. All the tweaks. Happily!

**Comment by rampd2** [comment]
possible duplicate of #5436
I think it is better to upvote the already existing feedback post instead of creating many.

**Reply by mgks** [reply]
oh, I searched but didn't found any earlier.

**Comment by s805310630** [comment]
Can only be created. If too many are created, it is not easy to find. It should be possible to set it to delete.

**Comment by fmohtadi99** [comment]
You mean adding "Delete" option is planned for future ???? Am I right?? EZ

**Comment by matheusmacedoap** [comment]
Esse recurso está no roadmap https://github.com/github/roadmap/issues/483 para o segundo trimestre (abril a junho)

**Comment by instance01** [comment]
I'm a little dumbfounded, are we really lacking the option to delete for... let me check... nearly 6 months now?!!

**Reply by matheusmacedoap** [reply]
exatamente!

**Reply by SavantHG** [reply]
exactly!

**Comment by 2percentsilk** [comment, marked as answer]
Thanks for your patience and feedback! We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/). Give it a try and let us know how it goes 😄

**Reply by SavantHG** [reply]
Thanks!
